controllers.py

```diff
@@ -318,6 +330,11 @@ def getPinned():
             "pinned": recipe.pinned,
             "user_name": first_name,
         }
+        if recipe.pinned:
+            recipe_dict["imageUrl"] = recipe.imageUrl
+            recipe.update_record(imageUrl=recipe.imageUrl)
```

Comment:
What's going on here? Of the pinned recipe rows, are we getting the existing imageUrl, adding it to recipe in local recipe_dict variable, and updating the recipe in the db with that same Url? 

This seems redundant and we should't be updating db rows every time it's queried